src/oaklib/cli.py

```diff
@@ -2286,23 +2287,65 @@ def descendants(
 @main.command()
 @click.argument("terms", nargs=-1)
 @click.option("-o", "--output", help="Path to output file")
+@output_type_option
 @click.option(
-    "--include-all-predicates/--no-include-all-predicates",
-    default=False,
-    show_default=True,
-    help="For formats that export only IS_A by default, this will include all possible predicates",
+    "-c",
+    "--config-file",
+    help="""Config file for additional params. Presently used by  `fhirjson` only.
+
+    Options:
+    code_system_id: For `fhirjson` only. The code system ID to use for identification on the server uploaded to.
+    See: https://hl7.org/fhir/resource-definitions.html#Resource.id
+
+    code_system_url: For `fhirjson` only. Canonical URL for the code system.
+    See: https://hl7.org/fhir/codesystem-definitions.html#CodeSystem.url
+
+    native_uri_stems: For `fhirjson` only. A comma-separated list of URI stems that will be used to determine whether a
```

Comment:
As you are documenting either a python structure or json structure, where the expected value is 

```json
"native_uri_stems": ["stem1", ... , "stemN"]
```

I don't think this should say comma-separated, unless you intend

```json
"native_uri_stems": "stem1, ..., stemN"
```